bitcoin_client_rs/tests/utils/mod.rs

```diff
@@ -25,27 +25,35 @@ impl RecordStore {
             }
             if let Some(resp) = exchange.strip_prefix("<=") {
                 let resp = Vec::from_hex(resp).expect(&format!("Wrong tests data {}: {}", i, resp));
-                store.queue.insert(command.clone(), resp);
+                store.queue.push((command.clone(), resp));
             }
         }
 
         store
     }
 }
 
-#[derive(Clone)]
```

Comment:
AtomicUsize cannot be cloned, that's why in tests RecordStore is cloned instead of the whole TransportReplayer